src/Integration/Binding/NuGetBindingOperation.cs

```diff
@@ -88,7 +78,9 @@ public bool InstallPackages(ISet<Project> projectsToBind, IProgress<FixedStepsPr
             var projectNugets = projectsToBind
                 .SelectMany(bindingProject =>
                 {
-                    var projectLanguage = ProjectToLanguageMapper.GetLanguageForProject(bindingProject);
+                    var projectToLanguageMapper = serviceProvider.GetMefService<IProjectToLanguageMapper>();
+                    var projectLanguages = projectToLanguageMapper.GetAllBindingLanguagesForProject(bindingProject);
+                    var projectLanguage = projectLanguages.FirstOrDefault();
```

Comment:
As in the other places, I'm only taking the first one if the code was using the Obsolete signature. 
I thought about changing it to multiple but I didn't want to touch the product code.